hydra-demo.cabal

```diff
@@ -65,11 +65,21 @@ common common-lang
 
 library
   import:          common-lang
-  exposed-modules: HydraRPS.Contract
+  exposed-modules: 
+    HydraRPS.OnChain
+    HydraRPS.OffChain
   build-depends:
+    , aeson
+    , bytestring
+    , containers
+    , lens
+    , plutus-contract
```

Comment:
If we are using `HydraRPS.OffChain` only for testing, should we move it from `src` to `test`, and respectively its dependencies (`plutus-contract`, etc.) from the `library` to the `test-suite`? This may improve build time in case one just wants to build the application without tests, since `plutus-contract` and its dependencies are quite heavy.

Reply:
It would also make it clear that `plutus-contract` is not directly applicable to working with Hydra Node API.